docker/base/Dockerfile.ubi

```diff
@@ -1,5 +1,5 @@
 # syntax = docker/dockerfile:experimental
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest as builder
+FROM redhat/ubi8-minimal:latest as builder
```

Comment:
Snyk won't scan images from the RH registry, ergo why this reference is changed to Docker Hub.